utils/client.js

```diff
@@ -1,7 +1,11 @@
-import ApolloClient from "apollo-boost";
-import { InMemoryCache } from "apollo-cache-inmemory";
+import ApolloClient from 'apollo-boost';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+
+const PORT = process.env.PORT || 3000;
+const path = process.env.ENV_NODE ==='development'?`http://localhost:${PORT}`:'https://vcars.herokuapp.com';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7125704)
```suggestion
const path = process.env.ENV_NODE ==="development"?`http://localhost:${PORT}`:'https://vcars.herokuapp.com';
```